### PR TITLE
Enable routing (fixes issue #11)

### PR DESCRIPTION
**Pull request by Gaurav0**
This adds a file named 'router.js' with a default implementation that can be used as the router to the Ember Twiddle default blueprint app

Fixes issue #11

### CODE REVIEW

File: app/lib/twiddle-resolver.js [hunk 1]
```diff
@@ -6,50 +6,44 @@ export default Resolver.extend({
   },
 
   resolveOther (parsedName) {
-    if (parsedName.fullName === 'router:main') {
```
Comment (joostdevries) [inline app/lib/twiddle-resolver.js]: `twiddle-resolver` doesn't do anything anymore as the application is now using the normal resolver

File: app/gist/new/route.js [hunk 2]
```diff
@@ -13,12 +13,16 @@ export default GistRoute.extend({
...
-      content: '<h1>Welcome to {{appName}}</h1><br><br>',
+      content: '<h1>Welcome to {{appName}}</h1><br><br>{{outlet}}<br><br>',
     }));
     model.get('files').pushObject(this.store.createRecord('gistFile', {
       filePath: 'controllers/application.js',
       content: 'import Ember from \'ember\';\n\nexport default Ember.Controller.extend({\n  appName:\'Ember Twiddle\'\n});',
     }));
+    model.get('files').pushObject(this.store.createRecord('gistFile', {
+      filePath: 'router.js',
+      content: 'import Ember from \'ember\';\nvar Router = Ember.Router.extend({\n  location: \'none\'\n});\n\nRouter.map(function() {\n});\n\nexport default Router;\n'
```
Comment (joostdevries) [inline app/gist/new/route.js]: I'd rather have an actual router with an address bar (although it will probably have to be readonly)

Comment (Gaurav0) [inline app/gist/new/route.js]: I can add an address bar if you'd like, but I'm not sure what you mean by an actual router?

File: app/lib/twiddle-resolver.js (hunk 1 above)
Comment (Gaurav0) [inline app/lib/twiddle-resolver.js]: I didn't know that. Should I delete it?

File: app/gist/new/route.js (hunk 2 above)
Comment (joostdevries) [inline app/gist/new/route.js]: sorry I mean actual location.

File: app/lib/twiddle-resolver.js (hunk 1 above)
Comment (joostdevries) [inline app/lib/twiddle-resolver.js]: That's cool.

File: app/gist/new/route.js (hunk 2 above)
Comment (Gaurav0) [inline app/gist/new/route.js]: Address bar added.

**Comment by stefanpenner** [comment]
YAY, good job @Gaurav0

**Comment by joostdevries** [comment]
:heart_eyes:

**Pull request merged**